HomeComponent test build fails

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

HomeComponent test build fails because of referencing a module that is not in the package.json.


### Reproduction Steps

1. Create new app with angular frontend
2. cd into angular folder
3. Run yarn
4. Run yarn test

### Expected behavior

Initial tests build and run without errors

### Actual behavior

Build fails with Error: Module not found: Error: Can't resolve '@abp/ng.theme.basic/testing' 

in ./src/app/home/home.component.spec.ts:2:0-70 
import { ThemeBasicTestingModule } from "@abp/ng.theme.basic/testing";

### Regression?

_No response_

### Known Workarounds

1. Add "@abp/ng.theme.basic/testing": "~7.0.3" to package.json
2. Run yarn
3. Run yarn test

### Version

7.0.3

### User Interface

Angular

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

### Other information

_No response_

--- a/templates/app-nolayers/angular/src/app/home/home.component.spec.ts
+++ b/templates/app-nolayers/angular/src/app/home/home.component.spec.ts
@@ -1,5 +1,4 @@
 import { CoreTestingModule } from '@abp/ng.core/testing';
-import { ThemeBasicTestingModule } from '@abp/ng.theme.basic/testing';
 import { ThemeSharedTestingModule } from '@abp/ng.theme.shared/testing';
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { NgxValidateCoreModule } from '@ngx-validate/core';
@@ -17,7 +16,6 @@ describe('HomeComponent', () => {
       imports: [
         CoreTestingModule.withConfig(),
         ThemeSharedTestingModule.withConfig(),
-        ThemeBasicTestingModule.withConfig(),
         NgxValidateCoreModule,
       ],
       providers: [

--- a/templates/app/angular/src/app/home/home.component.spec.ts
+++ b/templates/app/angular/src/app/home/home.component.spec.ts
@@ -19,7 +19,6 @@ describe("HomeComponent", () => {
         imports: [
           CoreTestingModule.withConfig(),
           ThemeSharedTestingModule.withConfig(),
-          //ThemeLeptonXModule.withConfig(),
           NgxValidateCoreModule,
         ],
         providers: [

--- a/templates/app/angular/src/app/home/home.component.spec.ts
+++ b/templates/app/angular/src/app/home/home.component.spec.ts
@@ -1,11 +1,11 @@
 import { CoreTestingModule } from "@abp/ng.core/testing";
-import { ThemeBasicTestingModule } from "@abp/ng.theme.basic/testing";
 import { ThemeSharedTestingModule } from "@abp/ng.theme.shared/testing";
 import { ComponentFixture, TestBed, waitForAsync } from "@angular/core/testing";
 import { NgxValidateCoreModule } from "@ngx-validate/core";
 import { HomeComponent } from "./home.component";
 import { OAuthService } from 'angular-oauth2-oidc';
 import { AuthService } from '@abp/ng.core';
+
 
 
 describe("HomeComponent", () => {
@@ -19,7 +19,7 @@ describe("HomeComponent", () => {
         imports: [
           CoreTestingModule.withConfig(),
           ThemeSharedTestingModule.withConfig(),
-          ThemeBasicTestingModule.withConfig(),
+          //ThemeLeptonXModule.withConfig(),
           NgxValidateCoreModule,
         ],
         providers: [

